code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm

```diff
@@ -140,26 +140,37 @@
 	sort_category = "Xenowear - Unathi"
 	flags = GEAR_HAS_DESC_SELECTION
 
-/datum/gear/augment/anchor
+/datum/gear/augment/autakh
 	display_name = "soul anchor"
 	description = "An implant that connects the Aut'akh to the Mesh."
```

Comment:
change to 
"A rune inscribed mirror or piece of glass placed behind the eyes. Believed to be the 'Window to the Soul' and house the concentrated spirit of an individual."